synapse/storage/schema/__init__.py

```diff
@@ -45,10 +45,11 @@
 Changes in SCHEMA_VERSION = 65:
     - MSC2716: Remove unique event_id constraint from insertion_event_edges
       because an insertion event can have multiple edges.
+    - Remove unused tables `user_stats_historical` and `room_stats_historical`.
 """
 
 
-SCHEMA_COMPAT_VERSION = 60  # 60: "outlier" not in internal_metadata.
+SCHEMA_COMPAT_VERSION = 61  # 60: "outlier" not in internal_metadata.
```

Comment:
nothing to add to the review, other than to say :tada: finally we get to use the `SCHEMA_COMPAT_VERSION` for its intended purpose!